packages/astro/src/actions/runtime/virtual/get-action.ts

```diff
@@ -11,7 +11,7 @@ import type { ActionAccept, ActionClient } from './server.js';
 export async function getAction(
 	path: string,
 ): Promise<ActionClient<unknown, ActionAccept, ZodType>> {
-	const pathKeys = path.replace('/_actions/', '').split('.');
```

Comment:
If there is a reason to split the pathKey based on . please let me know!

Reply:
Looks like a `.` could appear from here

https://github.com/withastro/astro/blob/8a1abcfd9d0507c68ede46d6f123002956c4340d/packages/astro/templates/actions.mjs#L50

So we likely still need to keep this. IIUC it's to support nested handlers in `actions.ts`: https://docs.astro.build/en/guides/actions/#organizing-actions

Reply:
Thanks for the great insight!